.travis/panic-test.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+
+# Check the unwrap and panic function presenting in src/ directory
+# of the repo. Testing suite is excluded. If such a function presents
+# return exit 1 for fail, otherwise return exit 0 for sucess.
+
+output=""
+panic_line=0
+
+for i in src/*; do
+    tmp_line=$(sed '/Unit Testing/q' $i | grep -e "unwrap()" -e "panic!(.*)" | awk 'END {print NR}')
+    if [ "$tmp_line" -gt 0 ]; then
+        tmp_output=$(sed '/Unit Testing/q' $i | grep -n -e "unwrap()" -e "panic!(.*)")
```

Comment:
I'd like to see a list of banned functions pulled out as a variable rather than inlined here.

Reply:
supported